src/cloudlabs/models.py

```diff
@@ -112,6 +112,7 @@ def default_setup_script(self, **kwargs):
              "AZURE_URL={dns_name}.ukwest.cloudapp.azure.com -p {port}:{port}"
              " web-app")]).format(**kwargs)
 
+
```

Comment:
Incidentally, you can view all PEP8 errors at https://travis-ci.org/UCL-CloudLabs/user-portal/jobs/259603673 - Travis checks these for us.